e2e/pages/adf/commentsPage.ts

```diff
@@ -18,7 +18,7 @@
 import { element, by } from 'protractor';
 
 import { Util } from '../../util/util';
-import { TabsPage } from './material/tabsPage';
+import { TabsPage } from '../../../lib/testing/src/lib/material/tabsPage';
```

Comment:
Shouldn't this import be something like: import { TabsPage } from '@alfresco/adf-testing';